src/main/java/mort/command/DeleteCommand.java

```diff
@@ -0,0 +1,32 @@
+package mort.command;
```

Comment:
I find it quite good that you broke down the commands into different classes, making each file much more manageable!